ecs-cli/modules/cli/compose/project/project_parseV3.go

```diff
@@ -210,6 +213,24 @@ func getWorkingDir(fileName string) (string, error) {
 	return filepath.Dir(pwd), nil
 }
 
+func convertToHealthCheck(healthCheckConfig *types.HealthCheckConfig) *ecs.HealthCheck {
+	ecsHealthcheck := &ecs.HealthCheck{
+		Command: aws.StringSlice(healthCheckConfig.Test),
+	}
+	// optional fields with defaults provided by ECS
+	if healthCheckConfig.Interval != nil {
+		ecsHealthcheck.Interval = adapter.ConvertToTimeInSeconds(healthCheckConfig.Interval)
+	}
+	if healthCheckConfig.Retries != nil {
+		ecsHealthcheck.Retries = aws.Int64(int64(*healthCheckConfig.Retries))
+	}
+	if healthCheckConfig.Timeout != nil {
+		ecsHealthcheck.Timeout = adapter.ConvertToTimeInSeconds(healthCheckConfig.Timeout)
+	}
```

Comment:
Do we not need to also set `StartPeriod`?

Reply:
See my comment in the PR description.

Reply:
Actually now I'm gonna add `StartPeriod` since I'm refactoring this method into `adapter/convert.go` so that it can be used to convert the ECS Params health check as well.

Reply:
Err, actually I'll wait to the next PR to add that.